eth2.0-implementers-calls/call_059.md

```diff
@@ -0,0 +1,212 @@
+
+# 3. Prater
+Video | [26:23](https://youtu.be/s017DQlsCCw?t=1583)
+
+**Danny** - Great. Yeah, so check that out any questions for show a or Proto on some of these test format updates. Great, and as it is a pre-release if you get into it, and there's something that totally didn't make sense in these new formats. Just let us know and we can iterate. Okay, next I want to talk about Prater a freeze on the call who has put together that PR and there's been some coordination. Although I'm not quite certain the status of launch and things like that. So afri or somebody else want to give us an update on where we stand.
+
+**Afr Schoe** - Yeah, sure. So I missed last call so kind of the back story was from how I understand it is we wanted to front run main-net with the test net. It has significantly more validators and we want to propose this test net project with bit more than 200,000 validators and instead of spamming the girly test networks a lot of deposits we decided to to pre-populate the Genesis with certain amount of validators and Proto was so kind to write a small go tool to generate test net canisters from a handful of seeds and this PR on the eth2 testnets repository is ready to go. So we took genesis type from the deposit contract on girly which was I don't know 11 days ago and had half a custom Genesis delays that puts it to I think next week on Wednesday, which should be the 11th or 16th of March and this is ready to be merged and I think unless anyone has any comments on that we can move on with this and look forward to Genesis next week. Yeah regarding genesis validators we have pre-populated gensis with 210,000 validators. I'm not sure where we stand regarding the regarding the just put this up here. We have a spreadsheet. I opened all the links except this one. So let's for each client teams 40K validators. We probably finished distributing the keys before the weekend, hopefully and then we can make sure that everyone is running their validators next week. I don't know Proto. Did you hand out any keys yet? 
```

Comment:
```suggestion
**Afr Schoe** - Yeah, sure. So I missed last call so kind of the back story was from how I understand it is we wanted to front run main-net with the test net. It has significantly more validators and we want to propose this test net project with bit more than 200,000 validators and instead of spamming the Goerli test networks a lot of deposits we decided to pre-populate the Genesis with certain amount of validators and Proto was so kind to write a small go tool to generate test net canisters from a handful of seeds and this PR on the eth2 testnets repository is ready to go. So we took genesis type from the deposit contract on Goerli which was I don't know 11 days ago and had half a custom Genesis delays that puts it to I think next week on Wednesday, which should be the 11th or 16th of March and this is ready to be merged and I think unless anyone has any comments on that we can move on with this and look forward to Genesis next week. Yeah regarding genesis validators we have pre-populated genesis with 210,000 validators. I'm not sure where we stand regarding the regarding the just put this up here. We have a spreadsheet. I opened all the links except this one. So let's for each client teams 40K validators. We probably finished distributing the keys before the weekend, hopefully and then we can make sure that everyone is running their validators next week. I don't know Proto. Did you hand out any keys yet? 
```